go.mod

```diff
@@ -1,3 +1,3 @@
-module github.com/eapache/go-resiliency
+module github.com/maximebeckman/go-resiliency
```

Comment:
For hopefully obvious reasons, I can’t merge this line into my version of the library.

Reply:
My bad, I need to test my change, and I didnt think it would been added to the PR 🙏